lib/broccoli/default-packager.js

```diff
@@ -944,8 +945,25 @@ module.exports = class DefaultPackager {
 
     let inputFiles = ['addon-test-support/**/*.js'];
 
+    // LEGACY: try booting up Testem -- will fail if a test framework hasn't
+    //         been loaded before "test-support"
     let footerFiles = ['vendor/ember-cli/test-support-suffix.js'];
 
+    // It is (and has been for a long while) up to the test-framework-adapter
+    // tool to call Testem's hookIntoTestFramework() method.
+    // Allowing the test framework tool do this is ideal, because
+    // ember-cli cannot possibly know when all the appropriate test-framework-providing
+    // script will be executed.
+    let hasModernTesting = false;
+    let emberQunitPath = resolvePackagePath('ember-qunit', process.cwd());
+    let emberQunitVersion = require(emberQunitPath).version.split('.');
+    if (parseInt(emberQunitVersion[0], 10) > 6) {
```

Comment:
this doesn't need to be exact, because it has no bearing on existing ember-qunit.

once Testem.hookInotTestFramework() is ran, it does not allow itself to be ran again -- so the ember-cli invocation is a no-op in older ember-qunits (older than the current beta)